scripts/generate_code.py

```diff
@@ -412,3 +412,20 @@ def glob(path, pattern):
 if not filecmp.cmp(new_reflection_file, original_reflection_file):
     shutil.move(new_reflection_file, original_reflection_file)
 shutil.rmtree(Path(reflection_path, temp_dir))
+
+# Python Reflection
+temp_dir = ".tmp"
+flatc(
+    ["-p", "--no-prefix"],
+    prefix=temp_dir,
+    schema="reflection.fbs",
+    cwd=reflection_path,
+)
+new_reflection_files = Path(reflection_path, temp_dir, "reflection")
+original_reflection_files = Path(
+    root_path, "python/flatbuffers/reflection"
+)
+if not filecmp.cmp(new_reflection_files, original_reflection_files):
+    shutil.rmtree(original_reflection_files)
+    shutil.move(new_reflection_files, original_reflection_files)
+shutil.rmtree(Path(reflection_path, temp_dir))
```

Comment:
Any way we can combine this logic with the C++ reflection generation code? Like abstract it out to its own flatc_reflection() function that can be invoked twice with the different flatc options?

Reply:
Done